support-frontend/assets/pages/contributions-landing/components/StripePaymentRequestButton/StripePaymentRequestButton.tsx

```diff
@@ -372,7 +382,7 @@ function setUpPaymentListenerSca(
 			});
 
 			if (props.contributionType === 'ONE_OFF') {
-				props
+				void props
```

Comment:
I think it would be good to add `catch` handlers for these, but it's quite a complex file so I wanted to minimise the number of changes that would show up in the compiled `js` for now

Reply:
My general strategy here was to `void` out all of the promises